docs/supported_apis/defaulting_to_pandas.rst

```diff
@@ -0,0 +1,34 @@
+Defaulting to pandas
+=======================================
+
+.. _defaulting-to-pandas-mechanism:
```

Comment:
What is this line needed for?

Reply:
I don't believe this line .. _defaulting-to-pandas-mechanism: does anything. Removed.